docs/README.md

```diff
@@ -2,10 +2,16 @@
 
 <p align="center">
   These docs are for
+  <a href="https://github.com/visgl/deck.gl/blob/9.0-release/docs/README.md">
+    <img src="https://img.shields.io/badge/deck.gl-v9.0-brightgreen.svg?style=flat-square" />
+  </a>
+  Looking for an old version?
+  <a href="https://github.com/visgl/deck.gl/blob/8.9-release/docs/README.md">
+    <img src="https://img.shields.io/badge/deck.gl-v8.9-brightgreen.svg?style=flat-square" />
+  </a>
```

Comment:
I'll just flag that the "these docs are for..." version appears to be outdated on each of the v8.7, v8.8, and v8.9 release branches. Perhaps we should find some way to validate this in the website deployment process, or remove the text? (Not to block this PR of course).

Reply:
For the old stuff we should probably just make a change to correct it on each of the branches